README.md

```diff
@@ -42,23 +42,35 @@ const { statusCode, data } = await anvilClient.fillPDF(pdfTemplateID, exampleDat
 
 console.log(statusCode) // => 200
 
-// Data will be the filled PDF raw byes
+// Data will be the filled PDF raw bytes
 fs.writeFileSync('output.pdf', data, { encoding: null })
 ```
 
 ## API
 
-### new Anvil({ apiKey })
+### new Anvil(options)
 
 Creates an Anvil client instance.
 
-* `apiKey` (String) - your API key from your Anvil organization settings
+* `options` (Object) - [Options](#options) for the Anvil Client instance.
 
 ```js
-const anvilClient = new Anvil({ apiKey })
+const anvilClient = new Anvil({ apiKey: 'abc123' })
 ```
 
-### Anvil::fillPDF(pdfTemplateID, payload)
+### Options
+
+Options for the Anvil Client. Defaults are shown after each option key.
+
+```js
+{
+  apiKey: <your_api_key>, // Required. Your API key from your Anvil organization settings
+  baseUrl: 'https://app.useanvil.com', // Optional. String setting the base URL for all client requests
+  userAgent: 'Anvil API Client/<version_number>' // Optional. String setting the User-Agent header in all requests
+}
```

Comment:
Not sure we want to put these front and center. They probably shouldnt be used. I'd be ok if they were undocumented

Reply:
Agreed.